src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/DataFlowPass.vb

```diff
@@ -615,13 +615,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             ' If the state of the slot is 'assigned' we need to clear it 
             '    and possibly propagate it to all the parents
+            If Me._tryState IsNot Nothing Then
```

Comment:
Turns out the similar code below does not need fixing because it is never used. The whole tryState deal is used to track conservative unassignemts in async capture analysis. However awaits unassign variables and do not unassign fields, so having parent variables and having tryState are mutually exclusivesituations.
I am going to remove the confusing code that tries to unassign parent variables in the tryState.
C# does not seem to have such code either.